src/EnergyPlus/HeatBalanceInternalHeatGains.hh

```diff
@@ -83,6 +83,13 @@ void SetupSpaceInternalGain(EnergyPlusData &state,
                             int RetNodeNum = 0 // for return air heat gains
 );
 
+// Multipliers are double counted for tank losses - these are the internal gains that need adjust for this
```

Comment:
It's not that the multipliers are double counted, but rather the loss should be distributed across the multiplied zones/spaces. Not sure how to say that more simply.